src/mastodon/mastodon.rs

```diff
@@ -1281,36 +1288,40 @@ impl megalodon::Megalodon for Mastodon {
         status: String,
         options: Option<&megalodon::PostStatusInputOptions>,
     ) -> Result<Response<MegalodonEntities::Status>, Error> {
-        let mut params = HashMap::<&str, String>::from([("status", status)]);
+        let mut params = HashMap::<&str, Value>::from([("status", serde_json::Value::String(status))]);
+        // let mut params = HashMap::<&str, Value>::from([("status", status)]);
```

Comment:
Can you delete this comment?
Is this required?

Reply:
oops. leftover from when I was testing the changes. 
New commit incoming

Reply:
a7e924b8b9bd97bd0656d520eff0abd2163f4c59